scss/bootstrap/overrides/style/_button-groups.scss

```diff
@@ -1,27 +1,16 @@
-.btn .caret {
-  margin: 0 5px;
-}
-
 .btn-group {
-
-  .btn {
-    box-shadow: none;
+  label,
+  label:hover {
     border-color: $input-border;
```

Comment:
i don't get this, why are now `<label>` elements getting a border color, but only if they are in a `.btn-group` element? When is that going to happen? Are we talking about something like
```html
<div class="btn-group">
  <label>some label</label>
</div>
```
?

Reply:
Yes, this is a current markup of leave request modal.

Reply:
I have refactored the Multiple Day Selector in the modal https://github.com/civicrm/civihr/pull/2536 so now if we use a standard Bootstrap markup, we don't need these extra styling.